NOMIS/nomis.go

```diff
@@ -51,25 +51,32 @@ type NameDetails struct {
 }
 
 var (
-	censusNationalStatistic = false
+	censusNationalStatistic = true
 	fileName                string
 	fullURLFile             string
 	title                   string
-	metaTitle               []int
 	num                     int
 )
 
 const (
-	censusYear    string = "2011"
-	censusVersion string = "1"
+	censusYear         string = "2011"
+	censusVersion      string = "1"
+	censusPersonalData string = "Sometimes we need to make changes to data if it is possible to identify individuals. This is known as 'statistical disclosure control'. In the 2011 Census, we:\u000a\u000a" +
```

Comment:
In the sdmx file that we download , all the text in the file have \u000a and thought this would be inline with that

Reply:
Ah, I see, in which case it would be a bit of a task converting them all. It will mean we will have the Unicode values and markdown values being used, which is inconsistent. But it isn't a breaking issue, and to resolve it would require a fair amount of work, so let's leave it be for now.